fastscroll/src/main/java/com/l4digital/fastscroll/FastScrollRecyclerView.java

```diff
@@ -116,6 +116,15 @@ public void setBubbleTextColor(@ColorInt int color) {
         mFastScroller.setBubbleTextColor(color);
     }
 
+    /**
+     * Set the fast scroll state change listener.
+     *
+     * @param onFastScrollStateChangeListener The interface that will listen to fastscroll state change events
+     */
+    public void setOnFastScrollStateChangeListener(OnFastScrollStateChangeListener onFastScrollStateChangeListener) {
```

Comment:
The interface name is fine, but I prefer the variable names to not include "on". Can you refactor this variable to `fastScrollStateChangeListener`?